### ISSUE TITLE
Get landmark roles right

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/mima#74]
Make all content resides in a landmark region.
- [x] Wrap `DivMap` with `<main>`
- [x] Wrap `MenuButton` with `<nav>`
- [x] Wrap `SearchButton` with `<form role="search">`
- [x] Wrap `LocatorButton` and `SaveButton` with `<section aria-label="control">` (to declare the `region` role)

> "Probably the most important rule for applying landmarks is to ensure all content resides in a landmark region to ensure no content is orphaned. This way a screen reader user can safely use landmark navigation without missing content." —[Rich Schwerdtfeger, Feb 12, 2013](https://www.tpgi.com/using-wai-aria-landmarks-2013/#comment-1247)
